config/params/mainnet_config.go

```diff
@@ -284,14 +286,17 @@ func FillTestVersions(c *BeaconChainConfig, b byte) {
 	c.AltairForkVersion = make([]byte, fieldparams.VersionLength)
 	c.BellatrixForkVersion = make([]byte, fieldparams.VersionLength)
 	c.CapellaForkVersion = make([]byte, fieldparams.VersionLength)
+	c.DenebForkVersion = make([]byte, fieldparams.VersionLength)
```

Comment:
Thanks for taking care all the configs!

Reply:
TODO: I think if a user-supplied yaml config doesn't specify a deneb fork version, we could fail to load it, because the zero-value for the version could conflict. I need to test this.